doc/navigating.rst

```diff
@@ -133,6 +133,44 @@ appear as a new tab in the settings dialog, where you can adjust its settings
 at 
 any time. Use **Account** > **Remove** to delete accounts. 
 
+File Manager Overlay Icons
+--------------------------
+
+The ownCloud sync client provides overlay icons, in addition to the normal file 
+type icons, for your system file manager (Explorer on Windows, Finder on Mac and 
+Nautilus on Linux) to indicate the sync status of your ownCloud files.
+
+The overlay icons are similar to the systray icons introduced above. They 
+behave differently on files and directories according to sync status 
+and errors. 
+
+The overlay icon of an individual file indicates its current sync state. If the
+file is in sync with the server version, it displays a green checkmark.
+
+If the file is ignored from syncing, for example because it is on your 
+exclude list, or because it is a symbolic link, it displays a warning icon.
+
+If there is a sync error, or the file is blacklisted, it displays an 
+eye-catching red X.
+
+If the file is waiting to be synced, or is currently syncing, the overlay 
+icon displays a blue cycling icon.
+
+When the client is offline, the icons remain in the state of the 
+last successful sync, as long as they are not touched locally. Once a file
+is edited locally, the icon will change to the 'waiting to sync' icon.
```

Comment:
The idea of showing a waiting to sync icon while offline is incompatible with the waiting to sync status bubbling up to parent directories between syncs, unless you'd like to do a local discovery on startup only to show what is waiting for sync vs the database state (since we can't afford a recursive scan of the whole tree).

Just hiding all overlay icons while offline/paused would allow things to be simpler and bug-free. The whole point of refactoring this code was to improve the reliability on OS X since the file status was often getting stuck in a sync or OK state.